### DISCUSSION TITLE
Hi, every time I start Forge, I get the following and it won't let me start:  ImportError: cannot import name 'Undefined' from 'pydantic.fields' (H:\Forge UI:\stable-diffusion-webui-forgevenv\libsite-packages\pydantic\fields.py). Press a key to continue . . .   Forge does not start

### DISCUSSION THREAD

**Discussion by DavidJLlanes** [lllyasviel/stable-diffusion-webui-forge#793] [General]
Hi, every time I start Forge, I get the following and it won't let me start:

ImportError: cannot import name 'Undefined' from 'pydantic.fields' (H:\Forge UI:\stable-diffusion-webui-forgevenv\libsite-packages\pydantic\fields.py).
Press a key to continue . . .

**Comment by a-ruzhentsev** [comment]
Hi, go to your stable-diffusion-webui folder, open requirements_versions.txt and add line pydantic==1.10.11

save requirements_versions.txt

Then run webui-user.bat

**Reply by josefwerner** [reply]
this worked for me!
